src/BinaryRewriting/Misc/CommandLineParser.cs

```diff
@@ -0,0 +1,1642 @@
+﻿
```

Comment:
This will make the second CommandLineParser we have in this repo... we should really clean that up at some future point.